src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -20,52 +20,53 @@ export class TransactionDetailsSummaryItem extends React.Component<
   TransactionDetailsSummaryItemProps
 > {
   render() {
-    const {
-      offerOverride,
-      order: {
-        itemsTotal,
-        mode,
-        totalListPrice,
-        shippingTotal,
-        shippingTotalCents,
-        taxTotal,
-        taxTotalCents,
-        buyerTotal,
-      },
-      ...others
-    } = this.props
-
-    console.log("FUXKKC", this.props.order)
-
+    const { offerOverride, order, ...others } = this.props
     return (
       <StackableBorderBox flexDirection="column" {...others}>
-        {mode === "OFFER" ? (
-          <>
-            {/* TODO: Seller's offer / Your offer (/ Buyer's offer? Will sellers see this component?) */}
-            <Entry label="Your offer" value={offerOverride || itemsTotal} />
-            {Boolean(itemsTotal) && (
-              <SecondaryEntry label="List price" value={totalListPrice} />
-            )}
+        {/* TODO: Seller's offer / Your offer (/ Buyer's offer? Will sellers see this component?) */}
+        {this.priceDisplayComponent(order, offerOverride)}
+        <Spacer mb={2} />
+        <Entry label="Shipping" value={this.shippingDisplayAmount(order)} />
 
-            <Spacer mb={2} />
-          </>
-        ) : (
-          <Entry label="Price" value={itemsTotal} />
-        )}
-        <Entry
-          label="Shipping"
-          value={this.formattedAmount(shippingTotal, shippingTotalCents) || "—"}
-        />
-        <Entry
-          label="Tax"
-          value={this.formattedAmount(taxTotal, taxTotalCents) || "—"}
-        />
+        <Entry label="Tax" value={this.taxDisplayAmount(order)} />
         <Spacer mb={2} />
-        <Entry label="Total" value={buyerTotal} final />
+        <Entry label="Total" value={order.buyerTotal} final />
       </StackableBorderBox>
     )
   }
 
+  private shippingDisplayAmount = order => {
```

Comment:
#minor 

public/private doesn't matter at all in react components since everything is essentially private by default (you never really interact with component instances outside of the components themselves). So leaving everything public is fine and the normal way of doing things.